fam_get_option.html

```diff
@@ -104,7 +108,7 @@ <h2>Input Arguments</h2>
 			<tr><td class="code">optionName</td><td>Name of the option to be retrieved.</td></tr>
 		</tbody>
 	</table>
-	The following optionNames are available:
+	The following option names are available:
```

Comment:
VERSION below is returning "0.0.1". This default is incorrect-- should match the version of the OpenFAM library